ulozto_downloader/downloader.py

```diff
@@ -1,15 +1,13 @@
 import os
-import sys
 import multiprocessing as mp
 import time
 from datetime import timedelta
 from types import FunctionType
 import requests
-import colors
 
 from .page import Page
-
-CLI_STATUS_STARTLINE = 6
```

Comment:
Nemám rád `*` importy. Tady je to sice docela safe, ale u větších projektů to vede k dost neviditelným chybám s přepisováním věcí.

Radši bych tady viděl něco jako:
```from . import constants, utils```
případně pokud se chceme vyhnout dlouhému `constants`, tak:
```from . import constants as const, utils```

A odpovídajícím způsobem upravit použití.